src/shared/components/case-viewer/case-viewer.component.ts

```diff
@@ -207,17 +218,14 @@ export class CaseViewerComponent implements OnInit, OnDestroy, AfterViewInit {
     window.location.hash = tabChangeEvent.tab.textLabel;
   }
 
-  private init() {
+  private init(): void {
     // Clone and sort tabs array
     this.sortedTabs = this.orderService.sort(this.caseDetails.tabs);
-
     this.caseFields = this.getTabFields();
-
-    console.log('caseDetails', this.caseDetails);
-
     this.sortedTabs = this.sortTabFieldsAndFilterTabs(this.sortedTabs);
 
-    console.log('sortedTabs', this.sortedTabs)
+    // EUI-3825. Build a FormGroup for use in the filtering.
```

Comment:
Does this comment add value? You've added a JsDoc to the method which covers it

Reply:
Good call. This was my initial comment before I added the JsDoc. :)

Reply:
Done. 👍